_steps/step7-funkcje.md

```diff
@@ -96,7 +107,13 @@ document.getElementById("inputKolorRzad2").onchange = zmianaKoloru;
 
 **Argumenty** pozwalają nam na korzystanie z pewnych wartości podczas wykonywania funkcji. Nazywamy to zwykle **przekazywaniem argumentów**.
 
-**Parametry** pozwalają ci na odnoszenie się do tych wartości wewnątrz funkcji podczas jej deklarowania.
+**Parametry** pozwalają Ci na odnoszenie się do tych wartości wewnątrz funkcji podczas jej deklarowania.
+
+<!-- Czy jest sens się rozpisywać o tym rozrónieniu? Mam wrażenie, że wielu programistów
```

Comment:
ten tutorial to szycie fragmentów moich notatek z fragmentami z Internetu po angielsku pisane wieczorami i w pociągu, jeśli uważasz, że nie warto rozpisywać się, to po prostu  można zrobić argumenty/parametry o dać jedną definicję, ale dodamy info, że w codziennej praktyce używane zamiennie